### PR TITLE
Fix planet url

### PR DESCRIPTION
**Pull request by landonreed**
This fixes the planet URL (which has recently been updated to HTTPS) and improves exception logging (including fixing the error message sent to the client).

### CODE REVIEW

File: src/main/java/com/conveyal/osmlib/Updater.java [hunk 1]
```diff
@@ -125,6 +131,10 @@ public String getDateString(long secondsSinceEpoch) {
     public List<Diff> findDiffs (String timescale) {
         List<Diff> workQueue = new ArrayList<Diff>();
         Diff latest = fetchState(timescale, 0);
+        if (latest == null) {
+            LOG.error("Could not find {}-scale updates from OSM!", timescale);
+            return new ArrayList<>();
```
Comment (abyrd) [inline src/main/java/com/conveyal/osmlib/Updater.java]: The canonical way to do this is with `Collections.EMPTY_LIST`.

File: src/main/java/com/conveyal/osmlib/VanillaExtract.java [hunk 2]
```diff
@@ -130,6 +131,9 @@ public void service(Request request, Response response) throws Exception {
                 tileSource.copyTo(sink);
                 response.setStatus(HttpStatus.OK_200);
             } catch (Exception ex) {
+                String badFormatMessage = "URI format: /min_lat,min_lon,max_lat,max_lon[.pbf|.vex] (all coords in decimal degrees)\n";
```
Comment (abyrd) [inline src/main/java/com/conveyal/osmlib/VanillaExtract.java]: Why is a string containing a newline being substituted into a log message? I think maybe badFormatMessage is intended to be the body of the response.

**Review by abyrd — CHANGES_REQUESTED** [review]
I think there might be a mistake in the final error reporting change, see comments.

**Comment by landonreed** [comment]
OK, thanks for that review, @abyrd. I have addressed your comments with the latest commit (9a5ec234ce7670842376e81bca25dd7cdabb9d34).

**Pull request merged**